Add CLI tool for easy push approval for administrators (#482)
finos/git-proxy: website/docs/usage.mdx

Once you have followed the [installation](installation) steps, run:

> To run Git Proxy

Reviewer: ```suggestion
```

git-proxy
```

> To run Git Proxy CLI

Reviewer: ```suggestion
To run GitProxy using the CLI:
```

Author: updated

```bash
git-proxy-cli
```

### Using [npx instead of npm](https://www.freecodecamp.org/news/npm-vs-npx-whats-the-difference/)

> You can also install & run `git-proxy` and `git-proxy-cli` in one step:

Reviewer: ```suggestion
You can also install & run `git-proxy` and `git-proxy-cli` in two steps:
```

Author: updated

> Git Proxy

Author: updated

```bash
npx -- @finos/git-proxy
```

> Git Proxy CLI

Author: updated
